files/tk_metrics

```diff
@@ -39,30 +40,42 @@ OptionParser.new do |parser|
   option.call(:"pe-version", 'The version of PE in use', opt_value, save_ident)
   option.call(:print, 'Print to stdout', opt_bool, save_ident, true)
   option.call(:ssl, 'Whether or not to use SSL when gather metrics', opt_bool, save_ident, true)
-
-  # This one is special because it's required
   option.call(:"metrics-type", 'Type of metric to collect', opt_value, save_ident)
 
   # Kinda pointless to include descriptions unless there's a help flag
   parser.on("-h", "--help", "Prints this help") { puts parser; exit }
 
 end.parse!
 
-raise('ERROR: Missing configuration option "metrics-type"') if config[:"metrics-type"].nil?
+# This is to support legacy from the time when passing --metrics-type was what
+# determined the filename of the config file, rather than the --config option.
```

Comment:
Is this important?  I'm inclined to just switch to `--config` not support just `--metrics-type` on its own